livekit-android-sdk/src/main/java/io/livekit/android/webrtc/SimulcastVideoEncoderFactoryWrapper.kt

```diff
@@ -165,6 +168,31 @@ internal class SimulcastVideoEncoderFactoryWrapper(
             val future = executor.submit(Callable { return@Callable encoder.implementationName })
             return future.get()
         }
+
+        override fun createNativeVideoEncoder(): Long {
+            val future = executor.submit(Callable { return@Callable encoder.createNativeVideoEncoder() })
+            return future.get()
+        }
+
+        override fun isHardwareEncoder(): Boolean {
+            val future = executor.submit(Callable { return@Callable encoder.isHardwareEncoder })
+            return future.get()
+        }
+
+        override fun setRates(rcParameters: VideoEncoder.RateControlParameters?): VideoCodecStatus {
+            val future = executor.submit(Callable { return@Callable encoder.setRates(rcParameters) })
+            return future.get()
+        }
+
+        override fun getResolutionBitrateLimits(): Array<VideoEncoder.ResolutionBitrateLimits> {
+            val future = executor.submit(Callable { return@Callable encoder.resolutionBitrateLimits })
+            return future.get()
+        }
+
+        override fun getEncoderInfo(): VideoEncoder.EncoderInfo {
+            val future = executor.submit(Callable { return@Callable encoder.encoderInfo })
```

Comment:
This is the main thing that was breaking the hardware encoder from working (and preventing H264 from working). EncoderInfo holds onto information needed for resolution alignment, which is used to adjust the resolution (only works on 16x16 aligned frames).